testing.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: page
+title: Testing
+permalink: /testing/
+---
+
+* TOC
+{:toc}
+
+## Introduction
+
+The more features your application has, the harder it is to test it manually. A
```

Comment:
I noticed you use the term "application" throughout this doc and in your main title. It's acceptable to shorten this to "app" which is the more common term for mobile (developers know that Flutter apps are for mobile devices).